static/js/components/property/level.js

```diff
@@ -145,6 +145,9 @@ class LevelProperty extends BaseComponent {
     this._upgradeProperty('value');
     this._upgradeProperty('disabled');
 
+    // Set the initial value so that both inputs are set properly.
+    this.value = parseInt(this.min, 10);
```

Comment:
Is there a chance that value could already be initialized here from the call to _upgradeProperty?

Reply:
Good call. Updated and should be safer now.